ui/src/containers/CreateVolume.js

```diff
@@ -514,19 +516,23 @@ const CreateVolume = (props) => {
 
               return (
                 <Form>
+                  <FormSection>
+                    <RequiredText>All * are mandatory fields</RequiredText>
```

Comment:
the text might need to be slightly changed to, please add translation as well.
`* Required fields`